pipelines/pingcap/tidb/latest/ghpr_build.groovy

```diff
@@ -206,4 +202,66 @@ pipeline {
             }
         }
     }
+    post {
+        always {
+            // TODO(wuhuizuo): put into container lifecyle preStop hook.
+            script {
+                // ============================================================
+                // name: 'PIPELINE_RUN_ERROR_CODE',
+                // 0 : 'success',
+                // 1 : 'checkout code fail',
+                // 2 : 'build fail',
+                // 3 : 'product test fail',
+                // 4 : 'flaky test error',
+                // 5 : 'fmt or lint fail',
+                // 6 : 'pipeline engine error',
+                // 7 : 'agent env error',
+                // 8 : 'other error',
+                // ============================================================                
+                def ciErrorCode = currentBuild.result == "SUCCESS" ? 0 : 2
+                def resultDownloadPath = "${FILE_SERVER_URL}/download/builds/pingcap/tidb-check/pr/${ghprbActualCommit}/centos7/tidb-server.tar.gz"
+                sh """
+                taskStartTime="\$(stat /proc/1 | grep 'Modify: ' | sed 's/Modify: //')"
```

Comment:
Is this the pipeline start time in millisecond?

Reply:
> Is this the pipeline start time in millisecond?
next line will transfer to unix milliseconds